File: home-tools-server/src/main/java/com/sunkl/hometoolsserver/utils/TimeUtils.java
package com.sunkl.hometoolsserver.utils;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class TimeUtils {
    final static String defaulDateTimeFormat = "yyyy-MM-dd HH:mm:ss";
    final static String defaulDateFormat = "yyyy-MM-dd";

    public static String getCurrentDateTime() {
        LocalDateTime localDateTime = LocalDateTime.now();
        return localDateTime.format(DateTimeFormatter.ofPattern(defaulDateTimeFormat));
    }

    public static String getYesterdayDate() {
        LocalDate today = LocalDate.now();
        return today.plusDays(-1).format(DateTimeFormatter.ofPattern(defaulDateFormat));
    }
}

File: home-tools-server/src/main/java/com/sunkl/hometoolsserver/utils/TimeUtils.scala
package com.sunkl.hometoolsserver.utils

import java.time.{LocalDate, LocalDateTime}
import java.time.format.DateTimeFormatter
import UdImplict._

object TimeUtils {
  private[utils] val defaulDateTimeFormat = "yyyy-MM-dd HH:mm:ss"
  private[utils] val defaulDateFormat = "yyyy-MM-dd"

  def getCurrentDateTime: String = {
    val localDateTime = LocalDateTime.now
    localDateTime.format(defaulDateTimeFormat.toDateTimeFromat)
  }

  def getYesterdayDate: String = {
    val today = LocalDate.now
    today.plusDays(-1).format(defaulDateFormat.toDateTimeFromat)
  }

  private def getCurrentLocalDateTime() = {
    LocalDateTime.now()
  }

  def getDateTimePlusMin(minNum: Int) = {
    this.getCurrentLocalDateTime().plusMinutes(minNum).format(defaulDateTimeFormat.toDateTimeFromat)
  }
}
